tmp/MOCK_COMMIT_HASH_MOCK_SESSION_ID.2022-04-17_21.log

```diff
@@ -0,0 +1,602 @@
+X|MOCK_COMMIT_HASH|MOCK_COMMIT_HASH|MOCK_COMMIT_HASH|MOCK_SESSION_ID|NA|2022-04-17T21:49:52-0400|openbb_terminal.parent_classes|menu|338|The command 'help' doesn't exist on the /alternative/ menu. - Traceback(most recent call last): - File openbb_terminal/parent_classes.py, line 334, in menu - self.queue = self.switch(an_input) - File cut/file/path/mock.py, line 1081, in __call__ - return self._mock_call(*args, **kwargs) - File cut/file/path/mock.py, line 1085, in _mock_call - return self._execute_mock_call(*args, **kwargs) - File cut/file/path/mock.py, line 1146, in _execute_mock_call - result = effect(*args, **kwargs) - File tests/openbb_terminal/alternative/test_alt_controller.py, line 102, in __call__ - raise SystemExit() - SystemExit
```

Comment:
I think we should make sure the test is not generating files.

Reply:
Good catch. Fixed!